Splitio-net-core-tests/Unit Tests/Cache/RedisImpressionCacheTests.cs

```diff
@@ -23,32 +21,10 @@ public void AddImpressionSuccessfully()
             var cache = new RedisImpressionsCache(redisAdapterMock.Object, "10.0.0.1", "net-1.0.2");
 
             //Act
-            cache.AddImpression(new KeyImpression() { feature = "test", changeNumber = 100, keyName = "date", label = "testdate", time = 10000000 });
+            cache.AddItem(new KeyImpression() { feature = "test", changeNumber = 100, keyName = "date", label = "testdate", time = 10000000 });
 
             //Assert
             redisAdapterMock.Verify(mock => mock.SAdd(key, It.IsAny<RedisValue>()));
         }
-
```

Comment:
no need for this test anymore?

Reply:
There was some dead code in the `RedisImpressionsCache` class for getting impressions from Redis. Since there is no Redis standalone configuration for the NET SDK I removed that method because it was never used. Because of that, I also removed this test